<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInfoBarWebReadersStatusSmallComponent</class>
 <widget class="QFrame" name="CInfoBarWebReadersStatusSmallComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Web readers status bar</string>
  </property>
  <layout class="QHBoxLayout" name="hl_CInfobarWebReaderStatusSmallComponent">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_SwiftDb">
     <property name="text">
      <string>DB</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_SwiftDb" native="true"/>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_DataReady" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Models">
     <property name="toolTip">
      <string>models</string>
     </property>
     <property name="text">
      <string>| mod.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Models" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Liveries">
     <property name="toolTip">
      <string>liveries</string>
     </property>
     <property name="text">
      <string>| liv.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Liveries" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Distributors">
     <property name="toolTip">
      <string>distributors</string>
     </property>
     <property name="text">
      <string>| dist.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Distributors" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Aircraft">
     <property name="toolTip">
      <string>aircraft</string>
     </property>
     <property name="text">
      <string>| ac.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_IcaoAircraft" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Airlines">
     <property name="toolTip">
      <string>airlines</string>
     </property>
     <property name="text">
      <string>| al.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_IcaoAirline" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Countries">
     <property name="toolTip">
      <string>countries</string>
     </property>
     <property name="text">
      <string>| ctry.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::CLedWidget" name="led_Countries" native="true"/>
   </item>
   <item>
    <spacer name="hs_InfoBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::CLedWidget</class>
   <extends>QWidget</extends>
   <header>gui/led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
